src/features/board/Board.tsx

```diff
@@ -15,11 +42,23 @@ export function Board () {
             <button onClick={() => dispatch(attack([1, 0]))}> Dragon attack Mage</button>
             <button onClick={() => dispatch(attack([1, 1]))}> Dragon attack Vampire</button>
             <div>
-                { opponentCards.map((card, i) => <Card key={`${card}${i}`} cardProperties={card} />) }
+                { opponentCards.map((card, i) => (
+                    <div>
+                        <Card key={`${card}${i}`} cardProperties={card} />
+                    </div>
+                )) }
             </div>
             <hr/>
             <div>
-                { playerCards.map((card, i) => <Card key={`${card}${i}`} cardProperties={card} />) }
+                { playerCards.map((card, i) => (
+                    <div
+                        onClick={() => { handleSelectCard(i) }}
+                        onContextMenu={handleContextMenu}
+                        style={{border: (selectedCardPosition === i) ? '1px solid black': ''}}
+                    >
+                        <Card key={`${card}${i}`} cardProperties={card} />
```

Comment:
This is a `map`, not a `forEach` so dont forget to return. (Otherwise it creates an array of undefined values).